common/sc_popen.cpp

```diff
@@ -163,10 +163,11 @@ std::tuple<pid_t, FILE*> sc_popen_argv(const std::vector<std::string>& strings,
         strings.begin(), strings.end(), std::string(),
         [](const std::string& a, const std::string& b) -> std::string { return a + (a.length() > 0 ? " " : "") + b; });
 
-    return sc_popen(std::move(commandLine), type);
+    return sc_popen_c(std::move(commandLine).data(), type.data());
 }
 
 std::tuple<pid_t, FILE*> sc_popen(std::string&& command, const std::string& type) {
+    command = "cmd /c " + command;
     return sc_popen_c(command.data(), type.data());
```

Comment:
`sc_open_c` is just the C-style version of `sc_open`, so they should have the same behavior (= spawn a process with the given arguments). If `cmd /c` is required, it should be explicitly added by the caller.

Reply:
Maybe I'm not understanding correctly? 
Running `String .unixCmd` in a shell is a [behavior defined in SC](https://github.com/supercollider/supercollider/pull/6837/files#diff-a15e8fe708dbb2bcd27256def5a360fbacc152340db2385f013b05af940fb8f8R675). On *nix we do [sh -c](https://github.com/supercollider/supercollider/pull/6837/files#diff-ddec1c4dde7311f60d467ba6918bc5abcfe4b4718638d305850f16d3263984e8R29-R30) so I think `cmd /c` matches the intended behavior on Windows?

Reply:
Of course, this would also apply to the Linux version. (`sc_popen` adds `/bin/sh -c` and forwards to `sc_popen_argv`.)

In either case, I would expect `sc_popen` to just run the given command as is.

If you are motivated, it would be cool to fix this on both sides. Otherwise just keep your solution, but maybe add a comment.

Reply:
Here's another idea: keep your fix, but rename `sc_popen` to something like `sc_popen_shell` in both implementations. Then it would be clear what the function does :)

Reply:
I renamed `sc_popen` to `sc_popen_shell`